pkg/api/schema/schema.proto

```diff
@@ -544,10 +570,26 @@ service ImmuService {
 		};
 	};
 
-	rpc UpdateAuthConfig (AuthConfig) returns (google.protobuf.Empty){} // DEPRECATED
```

Comment:
is none of the SDKs is using we may directly remove them?

Reply:
I will remove this in a new PR